frontend/src/components/footer.tsx

```diff
@@ -16,31 +16,90 @@ import OpenStaxURL from '../images/openstax-logo.png'
 
 const Funders = () => {
     return (
-        <Container bg='white'>
-            <Stack>
-                <Title order={4} className="fw-bold">Support from Scientific Agencies</Title>
-                <Flex justify='center' align='center' mx='auto'>
-                    <Anchor target="_blank" href="https://ies.ed.gov/">
-                        <Image alt="Institute of Education Sciences logo" src={IESLogoURL}/>
-                    </Anchor>
+        <Container style={{ backgroundColor: 'white', width: '100%', padding: '16px 32px 40px 32px' }}>
+            <Stack gap="md">
+                <Title
+                    order={2}
+                    style={{
+                        fontFamily: 'Helvetica Neue',
+                        fontSize: '36px',
+                        fontWeight: 700,
+                        lineHeight: '42px',
+                        letterSpacing: '-0.04em',
+                        textAlign: 'left',
+                    }}
+                >
+          Support from scientific agencies
+                </Title>
+          
+                <Flex
+                    style={{
+                        width: '100%',
+                        height: '256px',
+                        padding: '16px 0',
+                        gap: '0px',
+                        justifyContent: 'space-between',
+                        alignItems: 'center',
+                    }}
+                >
                     <Anchor target="_blank" href="https://www.nsf.org/gb/en">
-                        <Image alt="National Science Foundation logo" src={NSFLogoURL} />
+                        <Image 
+                            alt="National Science Foundation logo" 
+                            src={NSFLogoURL} 
+                            width={163.21} 
+                            height={120} 
+                            style={{ objectFit: 'cover', padding: '0px 32px' }} 
+                        />
                     </Anchor>
-                </Flex>
-
-                <Stack gap='md' mb='md'>
+    
+                    <Divider orientation="vertical" style={{ height: '200px', width: '1px', borderWidth: '1px' }} />
+    
+                    <Anchor target="_blank" href="https://ies.ed.gov/">
+                        <Stack
+                            align="center"
+                            gap="sm"
+                            style={{
+                                width: '462.66px',
```

Comment:
I would avoid using hardcoded widths (unless absolutely necessary) we want these to be somewhat responsive on mobile just in case they don't resize properly

Also keep in mind these `[mantine](https://mantine.dev/)` components have a lot of props for common css properties (ie above you replaced bg with a style={{background}} and here we have a gap `sm` property and you would be overwriting it with gap: 16px, theres also padding with the `p` prop